src/applications/claims-status/containers/ClaimsStatusApp.jsx

```diff
@@ -5,23 +5,47 @@ import PropTypes from 'prop-types';
 
 import backendServices from 'platform/user/profile/constants/backendServices';
 import { RequiredLoginView } from 'platform/user/authorization/components/RequiredLoginView';
+
 import { setLastPage } from '../actions';
 import ClaimsAppealsUnavailable from '../components/ClaimsAppealsUnavailable';
+import { isLoadingFeatures } from '../selectors';
 
 // This needs to be a React component for RequiredLoginView to pass down
 // the isDataAvailable prop, which is only passed on failure.
-function AppContent({ children, isDataAvailable }) {
+function AppContent({ children, featureFlagsLoading, isDataAvailable }) {
   const canUseApp =
     isDataAvailable === true || typeof isDataAvailable === 'undefined';
+  const shouldUseApp = canUseApp && !featureFlagsLoading;
```

Comment:
The variable name `shouldUseApp` isn't immediately clear to me: if a feature flag is loading, they can't really use the app (or can they?). 

The first thing that comes to mind for me is `isFeatureReady`. What thoughts do you have?

Reply:
Yeah if `!featureFlagsLoading` then we don’t want to load the app